src/main/resources/db/migration/V1_0__create_tables.sql

```diff
@@ -1,53 +1,63 @@
 CREATE TABLE visit
 (
```

Comment:
presumably you will be dropping the databases for these migration script merges

Reply:
Yes, taking the opportunity to consolidate the scripts whilst still in dev.